product-service/src/services/postgressProductService.ts

```diff
@@ -0,0 +1,74 @@
+import { Client } from 'pg';
```

Comment:
For transaction the Pool should be used

Reply:
@AntonVolyk Could you please explain why pool is required? As I understand we just need to be sure that we are using the same client for all statements within a transaction.

Reply:
@vohaha  As I understand correctly, the pool allows us to roll back the transaction via the Pool instance. 
Regarding the requirements, the stock should be created in case of successful operation of the product :
+1 (All languages) - Transaction based creation of product (in case stock creation is failed then related to this stock product is not created and not ready to be used by the end user and vice versa).